src/main/java/org/pac4j/jax/rs/annotations/Pac4JLogout.java

```diff
@@ -0,0 +1,33 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.core.context.Pac4jConstants;
+import org.pac4j.jax.rs.filter.ApplicationLogoutFilter;
+
+/**
+ * Identify the class or method as being filtered by {@link ApplicationLogoutFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JLogout {
+
+    /**
+     * @return value for {@link ApplicationLogoutFilter#setDefaultUrl(String)} (empty string means no redirect: instead
+     *         the annotated method will be executed).
+     */
+    String value() default "";
```

Comment:
Same as before: I don't think it should be the job of the logics but only of the users of the logics that can happen to have a null value. programmatically there is no reason for having a null value here.